docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/new-response-format.md

```diff
@@ -45,7 +45,7 @@ The Content API returns all the attributes of requested content wrapped inside a
       "title": "Article A"
       "relation": {
         "data": {
-          "id": "clkgylw7d000108lc4rw1bb6s"
+          "id": "14"
```

Comment:
```suggestion
          "id": "2"
```

I believe the `id` field here should have the value of the id of the "Article A" entry, right? So I suggest we use a different integer, so that users are not confused and believe that the they should use the id of "Category A" :)